app/frontend/src/lib/autocomplete.view.js

```diff
@@ -0,0 +1,105 @@
+import '../polyfill/after.polyfill';
+
+export const create = (container, input, onSelect) => {
+    if (!getRenderedList(container)) {
+        const ul = document.createElement('ul');
+        ul.setAttribute('id', 'location__listbox');
+        ul.setAttribute('role', 'listbox');
+        ul.setAttribute('tabindex', 0);
+
+        ul.classList.add('autocomplete__menu');
+        ul.classList.add('autocomplete__menu--overlay');
+
+        input.after(ul);
+
+        document.addEventListener('click', () => {
+            hide(container, input);
+        });
+
+        ul.addEventListener('click', (e) => {
+            onSelect(e.target.dataset.location);
+        });
+
+        input.addEventListener('keyup', e => {
```

Comment:
Is there a reason for `e` vs `(e)` on line 19?

Reply:
makes no difference, have changed for consistency